client/packages/invoices/src/Returns/InboundDetailView/DetailView.tsx

```diff
@@ -51,6 +51,20 @@ export const InboundReturnDetailView: FC = () => {
     },
   ];
 
+  const getNextItemId = () => {
```

Comment:
Is this logic that we have used in multiple places? Feels like the kind of thing that would be nice to have some unit tests around... and potentially easy to pull out/reuse?

Reply:
The other implementations have a `useNextItem` hook. I had a look and the logic is basically the same as what I've done here. However, I don't like calling it a hook, since it isn't one. Also, they'd made a whole new hook for every implementation, which I also don't like.

So what I've done now is just exported `getNextItem` as a separate utility function (in OuboundReturnsDetailView) and re-used it in Inbounds. e49f779cc

We should probably have a standardised pattern for these kind of "utility" functions as to where they go, etc., but they're not particularly widespread as far as I can see (there's just millions of hooks!!).